### PR TITLE
Spelling fixes

### PR DESCRIPTION
**Pull request by jsoref**
Followup to #123 

Generally, each commit is for a single word that was misspelled, possibly w/ many distinct misspellings.
You're welcome to squash commits.
You're also welcome to drop words.

I believe that all changes are to comments or equivalent content.
This means that I've excluded string changes (including debug output).
If this is taken, I can split the remainder into classes (debug output, local variables, apis, file renames).

### CODE REVIEW

**Comment by jsoref** [comment]
@RobDolin: hopefully this satisfies your needs.

In general, reviewing changes works better on small things. And given that multiple changes do impact individual lines, it becomes pretty hard to review a line when it's being changed in multiple places.

@rebornix: I can certainly squash (or someone else could squash) this series once someone determines that it's acceptable.

**Comment by RobStand** [comment]
We should hold off on merging these until the Blogger support is finished.

File: src/managed/OpenLiveWriter.Controls/LightweightControl.cs [hunk 1]
```diff
@@ -2191,7 +2191,7 @@ public virtual bool Unfocus()
 
         #region Accessibility
         /// <summary>
-        /// Returns a list of all the accessible controls contained in this control (includes decendents).
+        /// Returns a list of all the accessible controls contained in this control (includes decedents).
```
Comment (KirillOsenkov) [inline src/managed/OpenLiveWriter.Controls/LightweightControl.cs]: should be "descendants"

File: src/managed/OpenLiveWriter.CoreServices/HTML/HTMLDocumentHelper.cs [hunk 2]
```diff
@@ -101,7 +101,7 @@ public static string EscapePath(string html, string url, string newUrl)
 #endif
 
             // When an absolute path doesn't end with a file or a slash (i.e. http://www.realultimatepower.net)
-            // converting it to a Uri automatically adds the trailing slash (i.e. http://www.realtultimatepower.net).
+            // converting it to a Uri automatically adds the trailing slash (i.e. http://www.realultimatepower.net).
```
Comment (KirillOsenkov) [inline src/managed/OpenLiveWriter.CoreServices/HTML/HTMLDocumentHelper.cs]: Could you change this URL here and elsewhere to example.com while you're at it? Thanks!

**Comment by KirillOsenkov** [comment]
I've looked through all the changes and they look good. I've made 2 minor comments.
Thanks for the work @jsoref!

**Comment by martinwoodward** [comment]
Would you be able to squash into one commit as well?  Many thanks.

File: src/managed/OpenLiveWriter.CoreServices/HTML/HTMLDocumentHelper.cs (hunk 2 above)
Comment (jsoref) [inline src/managed/OpenLiveWriter.CoreServices/HTML/HTMLDocumentHelper.cs]: I've changed it, but, please note that err, the comment didn't actually make sense, so I've also added the trailing slash.

I can un-add the slash if you like, but..

**Comment by jsoref** [comment]
6c0e3c6 and 4b9836c are the changes I made for @KirillOsenkov. 449bc6c is the tip of that series.

I'm going to push a folded version next...

**Comment by jsoref** [comment]
Sigh. my tools keep botching the github interactions.

**Comment by jsoref** [comment]
@martinwoodward : fwiw, I squashed the commits...

**Comment by kathweaver** [comment]
This is showing as merged, but still open?  <shrug>  @ScottIsAFool?

**Comment by jsoref** [comment]
@kathweaver: I think the "merged" you saw was a cross comment from an unrelated msft repository about the fact that you have too many bots each of which asking contributors to sign what appears to be the same CLA.

This PR wasn't merged. I'm going to resubmit it (the branch only had one minor merge conflict 2 years on).

**Pull request closed without merging**